Content.Shared/Polymorph/Components/ChameleonProjectorComponent.cs

```diff
@@ -54,6 +57,12 @@ public sealed partial class ChameleonProjectorComponent : Component
     [DataField]
     public float MaxHealth = 100f;
 
+    /// <summary>
+    /// Popup shown to the user when they try to disguise as an entity inside a container.
+    /// </summary>
+    [DataField]
+    public LocId ContainerPopup = "chameleon-projector-inside-container";
```

Comment:
Unless these messages are intended to change across prototypes, there's no need to include them here, they can be inserted straight into the relevant system